packages/eas-cli/src/commands/update/index.ts

```diff
@@ -244,155 +237,73 @@ export default class UpdatePublish extends EasCommand {
     }
 
     let unsortedUpdateInfoGroups: UpdateInfoGroup = {};
-    let oldMessage: string, oldRuntimeVersion: string;
+    let oldRuntimeVersion: string;
```

Comment:
unused (looks like there's still some code related to republish that always throws due to this?)

Reply:
Done!